docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Global Description
+
+You would have a login page, a personal dashboard (number of images, etc).
+
+Homepage displays a list of users
+User's page displays a list of albums and in second time list of all pictures.
+If user doesn't have any album, just pictures, it would not display the album section.
+Album's page displays list of pictures belonging to the album
+
+# Uploader
+
+A user, when uploading can either populate an existing album or create a new one or just upload without album.
+There would be an uploader to upload multiple files at time with visual feedbacks (progression, error, success).
+Or you can create an album and then populate it.
+
+Images can be moved to an album later.
+
+Backend would create different images sizes optimized for web.
+
+# Album
+
+In an album, you have the list of pictures (thumbnail), album's properties. Album would have tags generated from all picture's tags inside it.
+
+Thumbnail of Album could be configured to be built based on random pictures inside the album or the last one or first one.
+
+Album can be private, kinda public (not listed but if you have the url you can access it), public.
+
+Album can be downloaded or not.
+
+## smart albums
+
+Albums that are generated automatically :
+- Uploads last 15 days
+- Favorites pictures (pictures could be faved)
+- Last batch uploaded
+- All public pictures
+
+# Pictures
+
+Pictures don't have a page strictly speaking, but there would be displayed full size in album when clicking on a thumbnail. You would a quite fullscreen with picture's properties info or just the image, two states would be toggled by clicking on the image.
+
+Pictures can have tags (that would be for a later use, maybe a search engine).
+
+# Preferences
+
+Per user you can have preferences for sorting, information display (exif or not, default privacy album, maybe image compression, possibility to download), then you can override it in album's preferences.
```

Comment:
```possibility to download``` 

Album ? Or image ? I can see the benefits of _blocking_ the download for an album, but not for images.

Reply:
For the album, it would make like a zip upon first request of download. Should make sure to make a routine as well (backend) to delete old archives. I'll precise it.

Reply:
Updated